package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -303,8 +306,7 @@ def parse(self, **kwargs):
         if atomtypes:
             attrs.append(Atomtypes(np.array(atomtypes, dtype=object)))
         else:
-            atomtypes = guessers.guess_types(names)
-            attrs.append(Atomtypes(atomtypes, guessed=True))
+            atomtypes = elements
```

Comment:
This needs to be upper case, since atomtypes are always upper case when guessed.